### PR TITLE
Update new-file-merge-conflicts.txt

### PR DESCRIPTION
**Pull request by xiulung**

### CODE REVIEW

**Review by experimentalgesture — APPROVED** [review]

**Comment by xiulung** [comment]
<font color="red">This text is red!</font>

> :sparkles: Love it!

> :rocket: **That was fast!**

> :gem: **Crystal clear**

> :bulb: **Great idea:**

> 📚 **Taught me something new**

Alt text here
Alt text here
Alt text here





https://graphite.dev/static/media/ramp.19cf984de4b92a5814fcde25a0d1a2cf.svg

```stl
solid cube_corner
  facet normal 0.0 -1.0 0.0
    outer loop
      vertex 0.0 0.0 0.0
      vertex 1.0 0.0 0.0
      vertex 0.0 0.0 1.0
    endloop
  endfacet
  facet normal 0.0 0.0 -1.0
    outer loop
      vertex 0.0 0.0 0.0
      vertex 0.0 1.0 0.0
      vertex 1.0 0.0 0.0
    endloop
  endfacet
  facet normal -1.0 0.0 0.0
    outer loop
      vertex 0.0 0.0 0.0
      vertex 0.0 0.0 1.0
      vertex 0.0 1.0 0.0
    endloop
  endfacet
  facet normal 0.577 0.577 0.577
    outer loop
      vertex 1.0 0.0 0.0
      vertex 0.0 1.0 0.0
      vertex 0.0 0.0 1.0
    endloop
  endfacet
endsolid
```

**Review by xiulung — COMMENTED** [review]
test

File: new-file-merge-conflicts.txt [hunk 1]
```diff
@@ -1,10 +1,14 @@
 new file fjkdlsaf 'f
 ds afd
- saf 
-kjdslafjdklsajfklds
-afjd fdsafdsa fdsafdsa
-sahfkhdklsahfklhdklsahf
-kdhsafhjdkshafjkhd
+fdjkslafjdklsaj fkd sjaf
+dsa fk
+dj saf d
```
Comment (xiulung) [inline new-file-merge-conflicts.txt:5]: test

**Review by xiulung — COMMENTED** [review]
test

**Pull request merged**